docs/rules/no-unused-class-name.md

```diff
@@ -0,0 +1,64 @@
+---
+pageClass: "rule-details"
+sidebarDepth: 0
+title: "svelte/no-unused-class-name"
+description: "disallow the use of a class in the template without a corresponding style"
+since: "v2.16.0"
```

Comment:
Please remove it as it will be added automatically.

```suggestion
```

After that you need to run `pnpm run update`.